### PR TITLE
Build update 

### PR DESCRIPTION
**Pull request by morggoth**
Hi!  I updated virtual box build for last release and added qemu-kvm build manifest.

### CODE REVIEW

**Comment by xen0l** [comment]
@morggoth thanks for your support. However, I have no way to test qemu vagrant box. Also, are you using it with Vagrant? If so, which plugin do you use?

**Comment by xen0l** [comment]
@morggoth I am testing the virtualbox build right now. If everything works, I will upload the box. I would also like to merge the PR, but don't have the option to test QEMU (macOS user). Could you split this PR into two: 1) one for virtualbox changes, 2) addition of QEMU/KVM?
EDIT: Virtualbox changes do not work for me. I will open a PR for that.

**Comment by xen0l** [comment]
I added changes that allowed me to build it. (I'm sorry I wasn't able to retain you as a committer, you deserve credits at least here!). Vagrant box has been published. Could you rebase the PR on top of latest changes, please? We are interested in the KVM thing.

**Comment by Mno-hime** [comment]
@morggoth Thanks for your contribution!

Does resulted QEMU Box work for you in libvirt provider? Can you successfully boot it?

With some tweaks (https://github.com/Mno-hime/oi-packer/tree/morggoth_kvm) I amble to build Vagrant box via QEMU builder, however the resulting Box fails to boot properly via `vagrant-libvirt` provider (https://github.com/vagrant-libvirt/vagrant-libvirt#installation) likely because the OS fails to find root filesystem. That's a common issue with illumos which happens when the VM setup changed (mostly disks, PCI) - and in this case the VM setup of builder VM and provider VM differ.

**Comment by Toasterson** [comment]
It does not boot for me either. We will need to find something to search the disk rather than hardcoding it in the boot_archive

**Comment by Toasterson** [comment]
The boot issue got fixed in illumos gate. I have merged your work with my rewrite to hcl2 configuration. in PR #4

File: hipster-qemu.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "variables": {
+    "vm_name": "oi-hipster",
+
+    "oi_version": "20181023",
+
+    "iso_url": "http://dlc.openindiana.org/isos/hipster/20181023/OI-hipster-text-20181023.iso",
```
Comment (pyhalov) [inline hipster-qemu.json:7]: Why do we use 20181023, not the latest?

Comment (Toasterson) [reply hipster-qemu.json:7]: Because this PR is that old :) look at #4 if you want the latest

**Comment by Toasterson** [comment]
This can now be closed as it is fixed by #4

**Pull request closed without merging**